cfme/intelligence/chargeback/assignments.py

```diff
@@ -67,24 +68,17 @@ class Assign(Updateable, Pretty, Navigatable):
             providers the rate is to be assigned.
 
     Usage:
-        enterprise = Assign(
-        assign_to="The Enterprise",
-        selections={
+        enterprise = appliance.collecions.assignments.instantiate("The Enterprise",
```

Comment:
typo

Reply:
Oops I never noticed that... WIll fix it asap!